src/js/tech/html5.js

```diff
@@ -176,32 +174,6 @@ class Html5 extends Tech {
     }
   }
 
-  // Make video events trigger player events
-  // May seem verbose here, but makes other APIs possible.
-  // Triggers removed using this.off when disposed
-  setupTriggers() {
-    for (let i = Html5.Events.length - 1; i >= 0; i--) {
-      this.on(Html5.Events[i], this.eventHandler);
-    }
-  }
-
-  eventHandler(evt) {
-    // In the case of an error on the video element, set the error prop
-    // on the player and let the player handle triggering the event. On
-    // some platforms, error events fire that do not cause the error
-    // property on the video element to be set. See #1465 for an example.
-    if (evt.type == 'error' && this.error()) {
-      this.player().error(this.error().code);
```

Comment:
I think I figured it out. Now the player listen for the error event on the tech the same way. This seems like a very good direction to take to abstract from the player.